rust/frontend/src/executor/distributed.rs

```diff
@@ -82,7 +82,7 @@ impl DistributedExecutor {
     pub async fn count(&mut self, plan: Count) -> Result<CountResult, ExecutorError> {
         let clients = self.clients(plan.scan.collection_and_segments.collection.collection_id)?;
         let res = (|| async {
-            choose_client(&clients)?
+            choose_client(clients.as_slice())?
```

Comment:
why this change?

Reply:
Don't know, it wasn't compiling and saying `expected &[], found &Vec<>`

Reply:
what version of rust are you on? because the PR passed tests

Reply:
strange 🤷